_docs-sources/refarch/access/how-to-auth-CLI/index.md

```diff
@@ -0,0 +1,9 @@
+# Authenticate via the AWS command line interface (CLI)
+
+A few important notes on authenticating via the CLI:
+
+1. You will need access keys. See [configuring your IAM user](../setup-auth/#step-2-configure-your-iam-user) for instructions.
+1. You will need an MFA token. MFA is required for the Reference Architecture, including on the CLI. See [configuring your IAM user](../setup-auth/#step-2-configure-your-iam-user) for instructions on setting up an MFA token.
+1. You will need to assume an IAM Role in most accounts. To authenticate to the security account, you only need your access keys and an MFA token. To authenticate to all other accounts (e.g., dev, stage, prod), you will need access keys, an MFA token, and the ARN of an IAM Role in that account to assume: e.g., to get read-only access to an account, you could assume the `allow-read-only-access-from-other-accounts` IAM Role. See the `cross-account-iam-roles` module for the default set of IAM Roles that exist in each account. Note that to be able to access an IAM Role xxx in some account yyy, your IAM User must be in an IAM Group that has permissions to assume that IAM Role. For example, to assume the `allow-read-only-access-from-other-accounts` IAM Role in the prod account, you must be in the `_account.prod-read-only` IAM Group. See [Configure other IAM users](http://localhost:3000/refarch/access/setup-auth/#step-3-configure-other-iam-users) for how you add users to IAM Groups.
```

Comment:
Not sure if this is the right link, you might need to try it locally

```suggestion
1. You will need to assume an IAM Role in most accounts. To authenticate to the security account, you only need your access keys and an MFA token. To authenticate to all other accounts (e.g., dev, stage, prod), you will need access keys, an MFA token, and the ARN of an IAM Role in that account to assume: e.g., to get read-only access to an account, you could assume the `allow-read-only-access-from-other-accounts` IAM Role. See the `cross-account-iam-roles` module for the default set of IAM Roles that exist in each account. Note that to be able to access an IAM Role xxx in some account yyy, your IAM User must be in an IAM Group that has permissions to assume that IAM Role. For example, to assume the `allow-read-only-access-from-other-accounts` IAM Role in the prod account, you must be in the `_account.prod-read-only` IAM Group. See [Configure other IAM users](./setup-auth#step-3-configure-other-iam-users) for how you add users to IAM Groups.
```

Reply:
Resolved in 4b6d06592